source/DownloadQueue.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+#include "common.h"
+#include <3ds.h>
+#include <vector>
+#include <fstream>
+#include <malloc.h>
+#include <unistd.h>
+
+#include "utils2.h"
+#include "utils.h"
+#include "ticket.h"
```

Comment:
Ah ok. For some reason, I thought best practice was to "hide" these in the .h

I didn't like it myself tbh